app/logic/validateNewEvent.py

```diff
@@ -43,7 +43,11 @@ def validateNewEventData(newEventData):
     event = Event.select().where((Event.eventName == newEventData['eventName']) &
                              (Event.description == newEventData['eventDescription']) &
                              (Event.startDate == newEventData['eventStartDate']))
+
     if event.exists():
+        if ignoreExist == True:

```

Comment:
This doesn't actually ignore anything. I'd probably flip this variable to be positive rather than negative. e.g., `checkExists` and default it to true. Then, your if statement can be `if checkExists and event.exists():`, which is pretty clear.